.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation Update
+
+## Description
+
+## Related Tickets & Documents
+
+- Related Issue #
+- Closes #
+
+## QA Instructions, Screenshots, Recordings
+
+_Please replace this line with instructions on how to test your changes, a note
+on the devices and browsers this has been tested on, as well as any relevant
+images for UI changes._
+
+### UI accessibility checklist
+_If your PR includes UI changes, please utilize this checklist:_
+- [ ] Semantic HTML implemented?
+- [ ] Keyboard operability supported?
+- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
+- [ ] Color contrast tested?
+
+_For more info, check out the
+[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._
```

Comment:
Please don't copy things directly from Forem/Oppia. The links for reference only. We don't need this.
Also, get rid of `UI accessibility checklist` for now.

Reply:
Done.